<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>331</width>
    <height>210</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Бирки</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="1" column="0">
    <widget class="QListWidget" name="lw_label">
     <property name="font">
      <font>
       <family>Tahoma</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="viewMode">
      <enum>QListView::IconMode</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lb_article">
     <property name="font">
      <font>
       <family>Tahoma</family>
       <pointsize>8</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Артикул&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>lw_label</sender>
   <signal>itemDoubleClicked(QListWidgetItem*)</signal>
   <receiver>Dialog</receiver>
   <slot>ui_select_label()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>196</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>555</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>ui_select_label()</slot>
 </slots>
</ui>
